docs/modules/configuring/con-configuring-kafkaservice-trust.adoc

```diff
@@ -0,0 +1,28 @@
+[id='con-configuring-kafkaservice-trust-{context}']
+= Trusting the proxied cluster's TLS certificates
+
+By default, when connecting to the proxied cluster using TLS, the proxy will use the platform's default trust store.
+This will usually work if the proxied cluster's TLS certificates were issued by a well-known public Certificate Authority (CA).
+It will not work if the proxied cluster's TLS certificates were issued by a private Certificate Authority.
+
+IMPORTANT: It is good practice to configure trust explicitly, even when proxied cluster's TLS certificates are signed by a public CA.
+
+.Example `KafkaService` configured to trust TLS certificates signed by any of the CAs in the `trusted-cas.pem` entry of the `ConfigMap` called `trusted-cas`
+[source,yaml]
+----
+kind: KafkaService
+metadata:
+  # ...
+spec:
+  bootstrapServers: kafka.example.com:9092
+  tls:
+    trustAnchorRef: # <1>
+      kind: ConfigMap # <2>
+      name: trusted-cas # <3>
+      key: trusted-cas.pem # <4>
+    # ...
+----
+<1> The `trustAnchorRef` object references a separate Kubernetes resource which contains the CA certificates to be trusted
+<2> The `kind` is optional and defaults to `ConfigMap`.
```

Comment:
shall we say what it's for and when to use it too?

Reply:
We have `kind` because we anticipate that eventually we might will support other kinds of resource. Because of the defaults it can be omitted. But including it explicitly makes it more obvious what's being referred to.

So I think it's better for users if they include it, because their resources are more understandable that way. But there's not much more to say about it right now, because we only support `ConfigMap`.